design/media_types.go

```diff
@@ -58,6 +58,45 @@ var workItem = MediaType("application/vnd.workitem+json", func() {
 	})
 })
 
+var pagingLinks = Type("pagingLinks", func() {
+	Attribute("prev", String)
+	Attribute("next", String)
+	Attribute("first", String)
+	Attribute("last", String)
+})
+
+var meta = Type("workItemListResponseMeta", func() {
```

Comment:
Is this `meta` specific to work items, then a qualified name would be better like `wiMeta`. Or if this is going to be generic, use the name as `ListResponseMeta`.